message_sender/factory.py

```diff
@@ -54,11 +54,14 @@ def _raw_send(self, py_data):
             'channel_data': channel_data,
         }
 
-        data = json.dumps(py_data)
+        data = json.dumps(data)
         r = self.session.post(self.api_url, auth=self.auth,
                               data=data, headers=headers)
         r.raise_for_status()
-        return r.json()
+        res = r.json()
+        return {
+            "message_id": res.get('result', {}).get('id')
+        }
```

Comment:
This is the only key that is being used at the moment. (the only place in the codebase that it's being used is in the task, and the task just uses the `message_id` field). Also, I think that this is all that we get back from Junebug anyway.